src/xbps/index.md

```diff
@@ -1,5 +1,33 @@
 # XBPS Package Manager
 
-The X Binary Package System (XBPS) is a package manager designed that was
-designed implemented from scratch and is extremely fast. XBPS is managed by the
-void-linux team and developed at <https://github.com/void-linux/xbps>.
+The X Binary Package System (XBPS) is a fast package manager that has been
+designed and implemented from scratch. XBPS is managed by the Void Linux team
+and developed at <https://github.com/void-linux/xbps>.
+
+Most general package management is done with the following commands:
+
+- [xbps-query(1)](https://man.voidlinux.org/xbps-query.1) searches for and
+   displays information about packages installed locally, or, if used with the
+   `-R` flag, packages contained in repositories.
+- [xbps-install(1)](https://man.voidlinux.org/xbps-install.1) installs and
+   updates packages, and syncs repository indexes.
+- [xbps-remove(1)](https://man.voidlinux.org/xbps-remove.1) removes installed
+   packages, and can remove cached package files.
```

Comment:
Mention it can remove orphan packages as well?

Reply:
Done.